doc/mk-site.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
-#
+
 export RUST_LOG="DEBUG"
 
 # clone in the legacy docs
-git clone -b gh-pages-legacy-html https://github.com/sl-sh-dev/sl-sh src/legacy/
+git clone -b gh-pages-legacy-html https://github.com/sl-sh-dev/sl-sh legacy/ || true
 
 # don't want git submodules
-rm -rf src/legacy/.git
-mkdir src/rust-docs
-
-cargo doc --features lisp-test --target-dir src/all-rust-docs
-cargo doc --no-deps --document-private-items --features lisp-test --target-dir src/slosh-rust-docs
+rm -rf legacy/.git || true
```

Comment:
TODO PC need to remove mk-site.sh and the build-doc function as things are now all done through the plugin.